lib/models/Episode.js

```diff
@@ -18,7 +18,20 @@ class Episode {
 
   static async getAll() {
     // implement getAll() method to return a list of Episodes with quotes
+    const { rows } = await pool.query(
+      `SELECT 
+      episodes.*, 
+      COALESCE(
+        json_agg(to_jsonb(quotes))
+        FILTER (WHERE quotes.id IS NOT NULL), '[]'
+    ) as quotes from episodes 
+      LEFT JOIN quotes on episodes.id = quotes.episode_id 
```

Comment:
Make sure to capitalize `AS` `FROM` and `ON` as well to keep your casing consistent!